src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/PathGradientBrush.cs

```diff
@@ -302,12 +304,14 @@ public ColorBlend InterpolationColors
                 // depending on how sane the input is. These checks exist to replicate the exact Desktop behavior.
                 int count = value.Colors.Length;
 
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                 if (value.Positions == null)
                     throw new ArgumentNullException("source");
                 if (value.Colors.Length > value.Positions.Length)
                     throw new ArgumentOutOfRangeException();
                 if (value.Colors.Length < value.Positions.Length)
                     throw new ArgumentException();
+#pragma warning restore CS2208
```

Comment:
Simlar question here.  Why don't we pass `value` here?

Reply:
For desktop compatability 

> // The Desktop implementation will throw various exceptions - ranging from NullReferenceExceptions to Argument(OutOfRange)Exceptions
// depending on how sane the input is. These checks exist to replicate the exact Desktop behavior.

Reply:
I think we should be ok with adding additional data to the exception.  I'm not sure why this code should be special in that regard.

Reply:
- I think we should be ok with adding additional data to the exception. I'm not sure why this code should be special in that regard.

That might be right, there could be some test failures in other env though, which we can make platform-specific. @ericstj please let me know is there any issue with updating argument name for an exception